sound/soc/sof/loader.c

```diff
@@ -123,7 +123,7 @@ int snd_sof_run_firmware(struct snd_sof_dev *sdev)
 					       "fw_version", 0444);
 		/* errors are only due to memory allocation, not debugfs */
 		if (ret < 0) {
-			dev_err(sdev->dev, "error: snd_sof_debugfs_buf_item failed\n");
+			dev_err(sdev->dev, "error: failed to add fw_version debugfs\n");
```

Comment:
can you remove the error: prefix while you're at it? We try to remove them one at a time when updating dmesg logs.

Reply:
looks like you removed the correction @cujomalainey ?

Reply:
Fixed :)